dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsApiServiceImpl.java

```diff
@@ -743,7 +745,8 @@ public ParsedRegistryID(String id) {
             }
             List<String> textSegments = Splitter.on('/').omitEmptyStrings().splitToList(id);
             List<String> list = new ArrayList<>(textSegments);
-            if (list.get(0).equalsIgnoreCase("#workflow")) {
+            String firstTextSegment = list.get(0);
+            if (WORKFLOW_PREFIX.equalsIgnoreCase(firstTextSegment) || SERVICE_PREFIX.equals(firstTextSegment)) {
```

Comment:
`equalsIgnoreCase` for SERVICE_PREFIX to be consistent with WORKFLOW_PREFIX? Or is that there for workflows for legacy reasons, and we won't allow different casing with services?

Reply:
Also, here because I can't comment on a line you haven't changed, the JavaDoc for this class should be updated to mention services.